README.md

```diff
@@ -18,121 +23,148 @@
 
-# 📖 BlogApp <a name="about-project"></a>
+# 📖 BLOG <a name="about-project"></a>
+
+In this milestone I created integration tests for all of the views used in the project. I covered the user stories (or user workflows) that I want my users to experience while taking into consideration the possible errors my users may cause.
 
 
-**BlogApp** 
-In this project, I set up tests on the controllers I recently added to my Blog app.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
+
 <details>
   <summary>Client</summary>
   <ul>
-    <li>Ruby on Rails</li>
+    <li><a href="https://guides.rubyonrails.org/engines.html">Embbed Ruby (ERB)</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li>Ruby on Rails</li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li>Postgresql</li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
 
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-
-- We can create, remove, update and delete posts. (CRUD)
-- It has user authentication.
-- It is well designed.
+- **Log into the application**
+- **Create a post for the application users**
+- **Interact with the posts (Like post or comment on Post)**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+<!--
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
+> Describe how a new developer could make use of your project.
 
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+ - Ruby and rails installed
+- PostgreSQL server installed also
 
-- Ruby
-- Ruby on Rails
-
+### Setup
 
+Clone this repository to your desired folder:
 
 ```sh
- gem install rails
+  mkdir blog-app
+  cd blog-app
+  git clone git@github.com:inspecta/Blog.git
 ```
 
+### Install
 
-### Setup
+Install this project with:
 
-Clone this repository to your desired folder:
+```sh
+  cd Blog
+  gem install
+
+### Usage
 
+To run the project, execute the following command:
 
 ```sh
-  $ git clone https://github.com/Bludidi/BlogApp.git
-  $ cd BlogApp
-  $ bundle install
-  $ rails db:create
-  $ rails db:migrate
-  $ rails s
+  rails server
 ```
 
-
 ### Run tests
 
 To run tests, run the following command:
+
 ```sh
-$ rspec.
+  bin/rails test test/models/article_test.rb
 ```
 
+<!--
+### Deployment
 
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly remove this text, it's a placeholder text guiding us how to make our README file professional